config/serverless.es.yml

```diff
@@ -69,3 +69,6 @@ xpack.searchInferenceEndpoints.ui.enabled: false
 
 # Search Notebooks
 xpack.search.notebooks.catalog.url: https://elastic-enterprise-search.s3.us-east-2.amazonaws.com/serverless/catalog.json
+
+# Search Homepage
+xpack.search.homepage.ui.enabled: true
```

Comment:
Hey @TattdCodeMonkey 
_question_: is this flag will be enabled by default for Search project on MKI (Cloud)?
If not, then the functional tests you added will probably fail: FTR can't override server arguments for Kibana/ES when running against real serverless projects (MKI), but tests still be working for local/ Kibana PR runs as here we start Kibana directly from build and pass arguments explicitly.

You can add `.tags(['skipOnMKI'])` on describe context to avoid running tests against MKI

Reply:
@dmlemeshko I had trouble with `createApiKeyForRole` in the FTRs when running against MKI locally so I went ahead and added `skipMKI` tag to the FTR tests for the feature flagged work.

Reply:
Could you share the problem you face with `createApiKeyForRole`, we can discuss it in `#appex-qa` channel.